### ISSUE TITLE
[Bug]: Incorrect url to docs for `sometimes` validations rule

### ISSUE THREAD

**Issue by Oleksandr-Moik** [laravel-idea/plugin#608]
### Bug description

When I opened the documentation with a keyboard shortcut, I accidentally noticed that the link was incorrectly generated.

Generated url by plugin - `https://laravel.com/docs/validation#rule-sometimes`
But expected and valid - `https://laravel.com/docs/validation#validating-when-present`

After analyzing the dock markup, was founded another rule with a different anchor:
- Multiple Of, `#multiple-of` - [line in doc](https://github.com/laravel/docs/blob/574147fec104d5b3e2377484fa2cc5d2269179b5/validation.md?plain=1#L854)
- Sometimes, `#validating-when-present`- [line in doc](https://github.com/laravel/docs/blob/574147fec104d5b3e2377484fa2cc5d2269179b5/validation.md?plain=1#L876)

### Plugin version

5.7.3.221

### Operating system

Linux

### Steps to reproduce

Click `Shift` + `F1` on `sometimes` (or `multiple_of`) validations rule

```php
'some_filed_1' => 'sometimes',
'some_filed_2' => 'multiple_of:',
```

**Comment by adelf** [comment]
Thank you. I thought they all have the same anchor as the rule name. I'll check them all and fix.

**Comment by adelf** [comment]
Fixed in the 6.0 release. Thanks.
